www/src/pages/en/deployment/docker.md

```diff
@@ -84,15 +84,16 @@ RUN \
 ##### BUILDER
 
 FROM --platform=linux/amd64 node:16-alpine3.17 AS builder
-ARG DATABASE_URL
+
 ARG NEXT_PUBLIC_CLIENTVAR
 WORKDIR /app
 COPY --from=deps /app/node_modules ./node_modules
 COPY . .
 
 # ENV NEXT_TELEMETRY_DISABLED 1
 
-RUN \
+RUN --mount=type=secret,id=DATABASE_URL \
+ export DATABASE_URL=$(cat /run/secrets/DATABASE_URL) && \
```

Comment:
This doesnt seem very nice if you have to add stuff here everytime?

Iirc 
- Environment are secure and not bundled in the image but only injected when you start the container.

- Args are unsafe and bundled in the image when it builds

Am I wrong?

Reply:
Yes right!

Reply:
btw here how i use that with github actions: 
https://github.com/upalatucci/biblioteca-nextjs/blob/main/.github/workflows/deploy.yml#L33

There is also a secret-file argument for that action but don't have experience with that

Reply:
In addition to that, I think this file (.env or a separate file) should be included in the .dockerignore as in Dockerfile we can find the command `COPY . .` . With that command we would copy the file with a secret in the dockerfile and this is another vulnerability